### DISCUSSION TITLE
Stored pictures?

### DISCUSSION THREAD

**Discussion by dimgod59** [ToTheBeginning/PuLID#39] [General]
Hello, I am discovering your service.
Concerning the photos that we send, are they stored on a server or everything is done locally with the GPU of the phone or computer ?

**Comment by ToTheBeginning** [comment]
If you are deploying the project to your local computer, then everything is on your local computer only. The huggingface demo we provided also will not store any picture.

**Comment by theboysdad1** [comment]
what about the replicate demo?  does that store the original, and or the predictions? thanks, and works amazing!  is this something i can use with automatic1111 or forge?
